InAppPurchasesAtlasAppServices/FoodieFolio/FoodieFolio/Views/RecipeDetail.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  RecipeDetail.swift
+//  FoodieFolio
+//
+//  Created by Mar Cabrera on 10/08/2023.
+//
+
+import SwiftUI
+
+struct RecipeDetail: View {
+    var recipe: Recipe
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 15) {
+                CircleImage(image: recipe.image ?? "")
+                    .padding(.top, 20)
+
+                Text(recipe.label ?? "Name not available")
+                    .font(.title)
+                    .padding(.bottom, 10)
+
+                let cuisineType = recipe.cuisineType.first ?? "Unspecified"
```

Comment:
Having a list of categories per recipe but then showing only one, is little bit confusing

Reply:
Yeah this was an issue with the API that I'm fetching data from. I've done some mapping in the App Services app and changed the code so it's only one type of cuisine returned